### DISCUSSION TITLE
Foreignkey attribute in schema

### DISCUSSION THREAD

**Discussion by russell310** [pydantic/pydantic#7452] [Question]
```
class Item(Base):
    id = Column(Integer, primary_key=True, index=True)
    title = Column(String, index=True)
    description = Column(String, index=True)
    owner_id = Column(Integer, ForeignKey("user.id"))
    owner = relationship("User", back_populates="items")


class ItemInDBBase(BaseModel):
    id: int
    title: str
    description: str
    owner_id: int

    class Config:
        from_attributes = True
```

What to do if I want owner_name without referencing model.

**Comment by sydney-runkle** [comment]
Hi - thanks for your question!

Could you give a bit more context for this question? I'm not sure what `owner_name` you're referring to. Which "model" are you referring to?

**Reply by russell310** [reply]
owner_name is referencing to User table which is backref as owner in Item table

```
class User(Base):
    id = Column(Integer, primary_key=True, index=True)
    name = Column(String, index=True)
```

**Reply by sydney-runkle** [reply]
Thanks for clarifying. I'm still not entirely sure I understand your question.

That being said, if you're using Pydantic and SQLAlchemy, you might want to consider using [SQLModel](https://sqlmodel.tiangolo.com/), a tool that leverages both.

If you could perhaps rephrase + give a bit more detail with your question, we might be able to help a bit more.

**Reply by russell310** [reply]
```
class ItemInDBBase(BaseModel):
    id: int
    title: str
    description: str
    owner_id: int
    owner_name: str         -> this is not present in item table, but in user table. How can I get it with the help of backref

    class Config:
        from_attributes = True
```
